UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/appsettings.Development.json

```diff
@@ -1,8 +1,15 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "AuthConfiguration": {
+        "MicrosoftOnlineLoginUrl": "https://login.microsoftonline.com/",
+        "TenantId": "9134ca48-663d-4a05-968a-31a42f0aed3e",
+        "ClientId": "6eb29be7-d5f5-4cab-b48f-96393e7b0695",
+        "Scope": "805be024-a208-40fb-ab6f-399c2647d334"
+
```

Comment:
Remove AuthConfiguration from appsettings as it is not used. Please do not commit values in PR.

Reply:
AuthConfiguration removed from appsettings

Reply:
As noted above - Please do not commit values in PR

"TenantId": "9134ca48-663d-4a05-968a-31a42f0aed3e"
"TenantId": ""

Reply:
appsettings.Development.json file corrected and pushed